HCCGo/app/js/navbar.js

```diff
@@ -4,6 +4,5 @@ navBar.controller('NavCtrl', ['$scope', '$log', 'navService', 'connectionService
    function($scope,$log,navService,connectionService, updaterService) {
    // This controller intended purely to manage navigation bar
    // No code beyond navigational controls should be used here
-   $scope.servicer = navService; 
-   
+   $scope.servicer = navService;
```

Comment:
Changes are in the first commit, second commit was me getting rid of a console.log that I had